### PR TITLE
Style portraits

### PR DESCRIPTION
**Pull request by mikeyrayvon**
- improves the styling and layout for the portraits (video, grid, & map)
- improves video frame sizing
- adds map zoom buttons & functionality (contrary to design: zoom buttons are horizontal. works better)
- some improvements to map interaction
- adds limit to map drag boundaries to not go into North/South grey areas. tried a few different approaches. these seems to work the best, albeit buggy. Oddly, google maps doesn't offer a native solution for this. disable `limitBounds()` to see the difference.

*improvements to map styling needed. there is a card for it.

### CODE REVIEW

**Comment by mikeyrayvon** [comment]
I recognize this is a big branch. Sorry. We're just running out of time on this... 😞

File: src/styl/responsive/mobile.styl [hunk 1]
```diff
@@ -176,19 +188,19 @@ $filters-height = 29px
 #support-form
   flex-wrap: nowrap
 
-::-webkit-input-placeholder // Chrome/Opera/Safari
+::-webkit-input-placeholder // @stylint ignore
```
Comment (varffs) [inline src/styl/responsive/mobile.styl:191]: what is all this about? is this not just generatable with a mixin of some sort?

**Review by varffs — APPROVED** [review]
Too big a PR to really be able to review properly IMO so going to not merge so @mixcas can also check.

Perhaps the limit bounds thing just needs a debounce on it's effect so it doesnt do the slightly buggy bobble?

File: src/styl/responsive/mobile.styl (hunk 1 above)
Comment (mikeyrayvon) [reply src/styl/responsive/mobile.styl:191]: this is stylint calling this prop invalid. you can add it to the valid props, or you can just ignore it like this. was very tired of seeing the error. if there’s a mixin then go ahead

**Comment by mikeyrayvon** [comment]
can you try to implement the debounce? @varffs

**Comment by mixcas** [comment]
IMO is better without the debounce/center/rezoom, all of that is really confusing. I'm going to revert back to before the refactoring and merge.

**Pull request merged**